geode-membership/src/main/java/org/apache/geode/internal/AvailablePort.java

```diff
@@ -547,12 +548,7 @@ public static void main(String[] args) {
 
     InetAddress addr = null;
     if (addrString != null) {
-      try {
-        addr = InetAddress.getByName(addrString);
-      } catch (Exception e) {
-        e.printStackTrace();
-        ExitCode.FATAL.doSystemExit();
-      }
+      addr = InetAddress.getByName(addrString);
```

Comment:
verified when address cannot be resolved the exception causes exit code 1 (on macos) ✓